lib/matplotlib/tests/test_basic.py

```diff
@@ -3,10 +3,17 @@
 
 import six
 
+import warnings
+
 from nose.tools import assert_equal
 
+from matplotlib.cbook import MatplotlibDeprecationWarning
 from matplotlib.testing.decorators import knownfailureif
-from pylab import *
+with warnings.catch_warnings():
+    warnings.filterwarnings('ignore',
+                            'The finance module has been deprecated in mpl 2',
+                            MatplotlibDeprecationWarning)
+    from pylab import *
```

Comment:
Does the warning makes any sense?
This will also create a merge conflict to master.